Doc/library/site.rst

```diff
@@ -16,7 +16,7 @@ import can be suppressed using the interpreter's :option:`-S` option.
 .. index:: triple: module; search; path
 
 Importing this module will append site-specific paths to the module search path
```

Comment:
```suggestion
Importing this module normally appends site-specific paths to the module search path
```
This suggestion is part of the next, but they cannot combined (GH rule).  The purpose is to move mention of `-S` to the next sentence.